exampleSite/utilities/README.md

```diff
@@ -59,6 +66,8 @@ Website = ""
 Facebook = ""
 Linkedin = ""
 Pronouns = ""
+Github = ""
+Twitter = ""
```

Comment:
Yeah, the only thing is, we need to document what all the possible fields are somewhere. Otherwise people have to read the code :)